### PR TITLE
Update msg_hash_ja.h

### PR DESCRIPTION
**Pull request by waitingmoon**

### CODE REVIEW

File: intl/msg_hash_ja.h [hunk 1]
```diff
@@ -2478,9 +2478,9 @@ MSG_HASH(MSG_USING_CORE_NAME_FOR_NEW_CONFIG,
 MSG_HASH(MSG_USING_LIBRETRO_DUMMY_CORE_RECORDING_SKIPPED,
       "Using libretro dummy core. Skipping recording.")
 MSG_HASH(MSG_VALUE_CONNECT_DEVICE_FROM_A_VALID_PORT,
-      "Connect device from a valid port.")
+      "デバイスを有効なポートから接続します")
```
Comment (bparker06) [inline intl/msg_hash_ja.h]: 接続してください*

File: intl/msg_hash_ja.h [hunk 2]
```diff
@@ -3921,10 +3921,10 @@ MSG_HASH(
     "YouTube"
     )
 MSG_HASH(MENU_ENUM_LABEL_VALUE_TWITCH_STREAM_KEY,
-      "Twitch Stream Key")
+      "Twitch配信キー")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_YOUTUBE_STREAM_KEY,
-      "YouTube Stream Key")
+      "YouTubeストリームキー")
```
[the remark this answers is not in the document]
Comment (waitingmoon) [reply intl/msg_hash_ja.h]: Japanese version Twitch uses '配信キー', but Youtube uses 'ストリームキー'
screenshot_2018-09-30 twitch
youtube_live26

[the remark this answers is not in the document]
Comment (waitingmoon) [reply intl/msg_hash_ja.h]: Excuse me, a confluct occured. but I don't know how to fix it.

**Comment by twinaphex** [comment]
I tried manually merging this. Now going to merge. Thanks @waitingmoon !

**Pull request merged**